stockflux-securities-master/src/components/ToolTip.css

```diff
@@ -0,0 +1,35 @@
+.tool-tip-container {
+  position: relative;
+  display: inline-block;
+}
+
+.tool-tip-container .tool-tip-message {
+  visibility: hidden;
+  background-color: var(--dark-background);
+  color: var(--text-grey-lighter);
+  text-align: center;
+  border-radius: 6px;
+  padding: var(--padding-small) var(--padding-medium);
+  position: absolute;
+  top: 0px;
+  right: 105%;
+}
+
+.tool-tip-container .tool-tip-message:after {
+  content: " ";
+  position: absolute;
+  top: 50%;
+  left: 100%;
+  margin-top: -5px;
```

Comment:
Is this negative margin intended?

Reply:
Since you requested my review again, I am asking again :D Is this negative margin intended? We should avoid negative margins/paddings/positions etc. if possible.

Reply:
Woops, I missed that comment but yes this negative margin is needed to keep the tooltip triangle arrow thing centred relative to the tool tip message.  I've tried to do it using positive bottom instead of negative top but this is the only way that seems to give the desired appearance. I know it is a bit hacky